.github/workflows/ci.yml

```diff
@@ -75,6 +75,7 @@ jobs:
     strategy:
       matrix:
         python_version: ['3.7']
+        ansible_version: ['==2.9.*', '']
```

Comment:
```suggestion
        ansible_version: ['==2.9.*', '==2.10.*', '==2.11.*', '']
```

Reply:
This won't work because there is no 2.11 release of ansible.

Reply:
so at least 2.10 in order to validate on different version